moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -457,22 +455,48 @@ const ProjectView = () => {
                           </Breadcrumbs>
                         </Box>
                       </Grid>
-                      <Grid item xs={1} md={1}>
-                        <Box
-                          className={classes.followDiv}
-                          onClick={() => handleFollowProject()}
-                        >
-                          <Tooltip title={isFollowing ? "Unfollow" : "Follow"}>
-                            {isFollowing ? (
-                              <BookmarkIcon className={classes.unfollowIcon} />
-                            ) : (
-                              <BookmarkBorderIcon
-                                className={classes.followIcon}
-                              />
-                            )}
-                          </Tooltip>
-                        </Box>
-                      </Grid>
+                      {!isProjectDeleted && (
+                        <Grid item xs={1} md={1}>
+                          <Box
+                            className={classes.followDiv}
+                            onClick={() => handleFollowProject()}
+                          >
+                            <Tooltip
+                              title={isFollowing ? "Unfollow" : "Follow"}
+                            >
+                              {isFollowing ? (
+                                <BookmarkIcon
+                                  className={classes.unfollowIcon}
+                                />
+                              ) : (
+                                <BookmarkBorderIcon
+                                  className={classes.followIcon}
+                                />
+                              )}
+                            </Tooltip>
+                          </Box>
+                        </Grid>
+                      )}
+                      {isProjectDeleted && (
+                        <Grid item xs={12}>
+                          <Box pb={2} pt={1}>
+                            <Alert severity="error">
+                              This project has been deleted and is no longer
+                              visible in Moped. If you need to restore a deleted
+                              project, please{" "}
+                              <Link
+                                href={
+                                  "https://atd.knack.com/dts#new-service-request/?view_249_vars=%7B%22field_398%22%3A%22Bug%20Report%20%E2%80%94%20Something%20is%20not%20working%22%2C%22field_399%22%3A%22Moped%22%7D"
+                                }
+                                target="new"
+                              >
+                                submit a Data &amp; Technology Services support
+                                request
```

Comment:
I would appreciate feedback on the wording here!

Reply:
I think it can be shorter: "This project has been deleted ~and is no longervisible in Moped~. If you need to restore...."